jest.config.ts

```diff
@@ -0,0 +1,10 @@
+import type { Config } from "jest";
+
+export default async (): Promise<Config> => {
+  return {
+    transform: { "^.+\\.ts?$": "ts-jest" },
+    testEnvironment: "node",
+    testRegex: "/tests/.*\\.(test|spec)?\\.(ts|tsx)$",
+    moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
```

Comment:
Since this is a `.ts` only project, I don't think you need to configure all of these extensions right?

```suggestion
    testRegex: "/tests/.*\\.(test|spec)?\\.ts$",
    moduleFileExtensions: ["ts", "json", "node"],
```